dbms/src/Common/UnifiedLogFormatter.cpp

```diff
@@ -31,30 +50,63 @@ void UnifiedLogFormatter::format(const Poco::Message & msg, std::string & text)
 
     // Timestamp
     {
+        if (enable_color)
```

Comment:
It would be better if we make `enable_color` as a template param and use `if constexpr (enable_color)`. But it does not block this PR